app/rest/transport_curl.cc

```diff
@@ -426,9 +445,7 @@ void BackgroundTransportCurl::CompleteOperation() {
 }
 
 void BackgroundTransportCurl::CheckOk(CURLcode code, const char* msg) {
-  if (code == CURLE_OK) {
-    return;
-  }
+  if (code == CURLE_OK) { return; }
```

Comment:
nit: There's a bunch of one-liners like this that scooped up the {}'s. Can we set it up to not one-line {} blocks? If we must one-line them, can we remove the {}'s?

I ran the formatter within google3 on a few test files and it *wasn't* turning the bodies of if statements into one liners if they were in curly brackets. Is this just an inconsistency in formatter rules? Which one is more correct?

Reply:
+1